``pytest-xdist`` is no longer used when running tests on PyPy.
